llvm/test/tools/UpdateTestChecks/update_mir_test_checks/multiple-defs.test

```diff
@@ -0,0 +1,6 @@
+## Check that update_mir_test_checks handles multiple definitions of the same
+## virtual register (after coming out of SSA form).
+
+# RUN: cp -f %S/Inputs/multiple-defs.mir %t.mir && %update_mir_test_checks %t.mir
+# RUN: FileCheck %t.mir < %t.mir
```

Comment:
We normally check the expected generated check lines using diff instead.

Reply:
Yeah, I know that's normal, but I'm not sure why it's better. In this case I wanted to demonstrate that the generated checks are broken.

Of course I can do it the "expected" way instead if that's what the reviewers really want.

Reply:
Done. I've added the "expected" output, but also run FileCheck to prove that the expected output really works.